source/_docs/installation/synology.markdown

```diff
@@ -2,78 +2,196 @@
 layout: page
 title: "Installation on a Synology NAS"
 description: "Instructions to install Home Assistant on a Synology NAS."
-date: 2016-04-16 11:36
+date: 2018-12-05 10:36
 sidebar: true
 comments: false
 sharing: true
 footer: true
 redirect_from: /getting-started/installation-synology/
 ---
 
-<p class='note warning'>
-Synology only provide Python 3.5.1, which is not compatible with Home Assistant 0.65.0 or later. Until Synology offer an updated version of Python, Home Assistant 0.64 is the most recent version that will be able to be installed. You can manually specify the version of Home Assistant to install, for example to install version 0.64.3 you would do `./python3 -m pip install homeassistant==0.64.3`
+<p class="note warning">
+Synology only provides [Python 3.5.1](https://www.synology.com/en-global/dsm/packages/py3k), which is not compatible with Home Assistant 0.65.0 or later. Until Synology offer an updated version of Python, Home Assistant 0.64 is the most recent version that will be able to be installed. If you want to run newer Home Assistant versions, you will need to make a new Python 3 package, which will be detailed in this guide. Otherwise you can manually specify the version of Home Assistant to install, for example to install version 0.64.3 you would do `./python 3 -m pip install homeassistant==0.64.3`
```

Comment:
...Until Synology offers ...

"...need to make a new Python 3 package" needs to be "...install the new Python 3 package from SynoCommunity"

Maybe remove the space between "python" and "3"; additionally, "/var/packages/python3/target/bin/pip3 install homeassistant==0.64.3" works.

Using "./python3" (ie "current directory" requires previous context to not change; I would suggest full pathnames so that the reader doesn't need to track down what directory he or she needs to be in.

Reply:
Thanks @chickenandpork, I did not pay much attention to that part since it was from the original guide, even though I should have.
I do wonder if I should include a part of the old guide at the end of the document under some header like "Previous guide" or something like that, then link to that from the warning note.

Reply:
Btw, is it better to link from "/volume1/@appstore/python3" or "/var/packages/python3" and if so why?

Reply:
First, it should be `/volume1/@appstore/python3`

I would suggest `/var/packages/python3` rather than `/volume1/@appstore/python3` because `volume1` is chosen by the user (I have two volumes, for example) so if the user chooses to install in `volume2`, the resulting path is a softlink from `/var/packages/python3/target` to `/volume2/@appstore/python3/` -- notice how `/var/packages/python3` remains correct, but `/volume1/` is now incorrect?  `/var/` links seem more predictable in this really rare case.

Reply:
For me the location does not really matter, as long as it is predictable.
Is there a big chance Synology might change "/var/packages/python3" to some different location in the future? If not, I would be using /var in a flash.